lib/katello/engine.rb

```diff
@@ -247,7 +247,7 @@ def self.with_katello_agent?
   end
 
   def self.remote_execution_by_default?
```

Comment:
Is this going to be removed by another PR?

Reply:
No. Maybe I should remove it here..